packages/commonwealth/client/scripts/helpers/snapshot_utils.ts

```diff
@@ -8,9 +8,10 @@ class SnapshotLazyLoader {
 
   private static async init() {
     if (!this.snapshot) {
-      this.snapshot = await import('@snapshot-labs/snapshot.js');
```

Comment:
this didn't work? we usually do this if the module is heavy and we dont want to load it eagerly

Reply:
@masvelio I will revert it and try, but I was getting errors with this flow. Will post the errors here if they are still happening.

Reply:
@masvelio This is the error I'm getting. Have tried to change things around and retain the await, but nothing I've tried seems to be working. If you have any suggestions let me know. 
![Screenshot 2024-07-08 at 9 23 43 AM](https://github.com/hicommonwealth/commonwealth/assets/69872984/15064248-1415-406d-bc8e-d13511724bce)

Reply:
thats fine then

Reply:
@masvelio Okay thanks

Reply:
@Israellund I think you might have the import path wrong here.  

You might be able to fix this by doing a console.log() of what was returned from await import('@snapshot-labs/snapshot.js') to see if you need to deconstruct it or maybe if it just returns the the default constructor as the default object.

Just trying to give advice - not sure if it's really worth fixing this